doc/source/devel/devdiscuss.rst

```diff
@@ -21,7 +21,6 @@ progress.
 
     spm_use
     modified_images
-    data_pkg_design
     data_pkg_discuss
     data_pkg_uses
```

Comment:
Should we go ahead and drop these two as well?

Reply:
Yes, they can all go.